mne/channels/montage.py

```diff
@@ -634,6 +635,8 @@ def _set_montage(info, montage, match_case=True):
         montage = make_standard_montage(montage)
 
     if isinstance(montage, DigMontage):
+        _check_option('on_missing', on_missing, ('raise', 'ignore'))
```

Comment:
on_missing='warning' should be supported to be consistent with other occurence of this parameter in the rest of the  code